workers/varcode_annotator.py

```diff
@@ -132,7 +132,10 @@ def get_varcode_annotations(genotypes, vcf_id, ensembl_release_num):
         best_effect = variant.effects().top_priority_effect()
         gene_name = best_effect.gene_name
         transcript = best_effect.transcript_id
-        notation = best_effect.short_description
+        if best_effect.__class__.__name__ == "Intragenic":
```

Comment:
thanks @ihodes! I did it this way because there are a few other classes of variants that implement Intragenic but have their short_description implemented. AFAIK, this is the only class that doesn't have that field.